travertino/src/travertino/compat.py

```diff
@@ -1,16 +1,14 @@
 from functools import cache
 from importlib.metadata import PackageNotFoundError, metadata
 
-from packaging.version import Version
-
 
 @cache
 def _toga_lt_5():
     try:
-        toga_version = Version(metadata("toga-core")["version"])
+        toga_version = metadata("toga-core")["version"]
     except PackageNotFoundError:  # pragma: no cover
         # If Toga isn't installed at all, we shouldn't invoke the backwards
         # compatibility shim; any exceptions must be from other sources.
         return False
     else:
-        return toga_version < Version("0.5.0")
+        return tuple(int(v) for v in toga_version.split(".")[:2]) < (0, 5)
```

Comment:
```suggestion
        major, minor, *_ = toga_version.split(".")
        return (int(major), int(minor)) < (0, 5)
```

I know this is just for a shim and won't even be around for long, but I do so prefer the [readability of multiple assignment over slicing](https://treyhunner.com/2018/03/tuple-unpacking-improves-python-code-readability/), when feasible.